src/shared/components/plots/StackedBarPlot.tsx

```diff
@@ -285,10 +303,16 @@ export default class StackedBarPlot extends React.Component<IStackedBarPlotProps
     }
 
     @computed get chartExtent() {
-        const miscPadding = 100; // specifying chart width in victory doesnt translate directly to the actual graph size
+        let miscPadding = 100; // specifying chart width in victory doesnt translate directly to the actual graph size
         if (this.data.length > 0) {
-            const numBars = this.data[0].counts.length;
-            return this.categoryCoord(numBars - 1) + 2*this.categoryAxisDomainPadding + miscPadding;
+            let numBars = 0;
+            if (this.props.stacked) {
+                numBars = this.data[0].counts.length;
+            } else {
+                numBars = _.sumBy(this.data, x => x.counts.length);
+                miscPadding = miscPadding - this.categoryCoord(this.data.length / 2);
```

Comment:
this is probably related to the formula in categoryAxisDomainPadding? can you explain? it might be helpful to pull out `@computed get SOMENAME() { return this.categoryCoord(this.data.length / 2); }` and reuse it and `SOMENAME` explains something about the rationale behind it